content/games/factions/companies.md

```diff
@@ -205,3 +236,9 @@ Unerring (3)
 
 Vicious (3)
 : To-hit rolls of 6 inflict two hits.
+
+| Taunt                      | Check | Requirements                                                                                                         | Effect                                                                                                                                                                                                                                                                                                                                           |
+| :------------------------- | :---: | :------------------------------------------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Our Streets, Your Skulls!  |  6+   | Street Toughs only. Must be within 8" of a Bloodied or Shaken enemy Group.                                           | This Group loses the Defensive trait and becomes Reckless against Shaken and Bloodied Groups. If this Group becomes Bloodied, they must test Resolve immediately; if they pass, they become Unfeeling.                                                                                                                                           |
+| Even Our Dogs Piss on You! |  7+   | Neighborhood Defenders only. Must be within 8" of an enemy Group Shaken by Kennel Hounds and not Bloodied or Shaken. | This Group loses the Defensive trait and becomes Vicious. If this Group becomes Bloodied or Shaken, they must test Resolve immediately; if they fail, they Rout.                                                                                                                                                                                 |
+| We'll be Home By Dinner!   |  8+   | First Citizens only. Must be within 12" of a Group they command.                                                     | This Group becomes Reckless. At the beginning of every turn, roll 2d6. If the result is less than the number of turns since they Proclaimed this taunt, they become Shaken. If the result is greater, they may reroll to activate for Reckless. If the result is equal, they become Unfeeling and Vicious and do not roll for this effect again. |
```

Comment:
```suggestion
| We'll be Home By Dinner!   |  8+   | First Citizens only. Must be within 12" of a Group they command.                                                     | This Group becomes Reckless. At the beginning of every turn, roll 2d6. If the result is less than the number of turns since they Proclaimed this taunt, they become Shaken. If the result is greater, they continue their activation as normal. If the result is equal, they become Unfeeling and Vicious and lose this Taunt's effect. |
```